packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -426,27 +469,40 @@ contract L1StandardBridge_DepositETHTo_Test is PreBridgeETHTo {
     ///      EOA or contract can call depositETHTo.
     ///      ETH ends up in the optimismPortal.
     function test_depositETHTo_succeeds() external {
-        _preBridgeETHTo({ isLegacy: true });
+        _preBridgeETHTo({ isLegacy: true, value: 600 });
         l1StandardBridge.depositETHTo{ value: 600 }(bob, 60000, hex"dead");
         assertEq(address(optimismPortal).balance, 600);
     }
 }
 
+contract L1StandardBridge_DepositETHTo_TestFail is Bridge_Initializer {
+    /// @dev Tests that depositETHTo reverts with custom gas token.
+    function test_depositETHTo_customGasToken_reverts() external {
```

Comment:
This could be good to fuzz to get the case of value and also zero value

Reply:
I thought we'd just revert for all values with custom gas token?